streamline/forms.py

```diff
@@ -72,6 +72,8 @@ def get_form(self):
         Return form object. Depending on the HTTP verb, this method returns
         either an unbound (GET) or a bound (all other verbs) form.
```

Comment:
"Once a form object is created, this method will keep returning the previously created instance."